packages/mobile/src/screens/profile-screen/MessageLockedButton.tsx

```diff
@@ -37,13 +23,12 @@ export const MessageLockedButton = (props: MessageLockedButtonProps) => {
 
   return (
     <Button
-      style={styles.root}
-      noText
-      title={messages.inboxUnavailable}
-      icon={IconMessageLocked}
-      variant={'common'}
+      iconRight={IconMessageLocked}
+      variant='secondary'
       size='small'
+      style={{ opacity: 0.4 }}
```

Comment:
do we have a disabled button style to cover this?

Reply:
yes we should, let me confirm the pattern here because i think we still want the on-press to happen to show a popup.